### ISSUE TITLE
Form groups in overlays are not programmatically associated with their group label [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#252]
The “Select your county”, “Gender identity”, “Age range”, “Race”, and “Ethnicity” overlays include form groups with an overall group label, but the group label is not announced upon focusing a control in the form group. For example, the “35-49” form control in the “Age range” overlays is simply announced as “35-49”.

![overlay-List-Age-Range](https://user-images.githubusercontent.com/7034772/92040342-a7620780-ed44-11ea-9df2-e1ce08e46b59.png)

**Remediation Guidance:**

**Comment by floridemai** [comment]
@jonnyjames This has now been addressed

**Comment by jonnyjames** [comment]
@floridemai This issue is now fixed. I am closing this issue.
